app/apis/api_application.py

```diff
@@ -83,6 +96,14 @@ def post(self):
                                description=description)
             db.session.add(aobj)
             db.session.flush()
+        if Auth.enabled():
+            user_id = get_jwt_identity()
+            roleObj = ApplicationUserRole(
+                application_id=aobj.application_id,
+                user_id=user_id,
+                role=Role.owner.name)
+            db.session.add(roleObj)
+            db.session.flush()
```

Comment:
Don't we need check the existence?

Reply:
This method creates a new application. The `application_user_roles` record associated with this application should not exist.

Reply:
No.
If we specify the existing application name, just add the service. I am not sure the current spec is appropriate or not but this process raises error in the case I mentioned.

Reply:
fixed.
https://github.com/rekcurd/drucker-dashboard/pull/49/commits/199c469f115cb8fa6ad4e0624c8e17ed75016a31